client/data/store.js

```diff
@@ -23,13 +24,15 @@ export const initStore = () =>
 			charges: charges.reducer,
 			timeline: timeline.reducer,
 			disputes: disputes.reducer,
+			multiCurrency: disputes.reducer,
```

Comment:
I think this should be `multiCurrency.reducer` :)

Reply:
Ha, another copy/paste issue. Fixed in https://github.com/Automattic/woocommerce-payments/pull/1856/commits/491b258ba28a0cde79b1d8f9055b32605591fe5c